x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/enrichments/index.test.ts

```diff
@@ -30,7 +31,14 @@ jest.mock('./enrichment_by_type/user_risk', () => ({
   ...jest.requireActual('./enrichment_by_type/user_risk'),
   getIsUserRiskScoreAvailable: jest.fn(),
 }));
+
+jest.mock('./enrichment_by_type/asset_criticality', () => ({
+  ...jest.requireActual('./enrichment_by_type/asset_criticality'),
+  doesAssetCriticalityIndexExist: jest.fn(),
+}));
```

Comment:
To me this is an indication that `doesAssetCriticalityIndexExists` (sic) should live independent to the enrichments themselves: we're trying to mock a file that we also want to test.